renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "automerge": true,
+  "major": {
+    "automerge": false
+  },
+  "timezone": "UTC",
+  "extends": [
+    "config:base",
+    ":maintainLockFilesWeekly"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 8am on Tuesday"
+  },
+  "packageRules": [
+    {
+      "packagePatterns": [
+        "*"
+      ],
+      "rangeStrategy": "pin"
```

Comment:
@wagnerand @Rob--W at the moment the dependencies are not pinned (https://github.com/mozilla/extension-activity-monitor/blob/5a2376a8fd306c96db1de9fba29b44f87a055100/package.json#L37-L50), do we want to make them pinned?

Reply:
I think we should pin them, yes. Renovate will submit PRs for updates and only auto-merge them after 7 days of stability and for minor updates only. That should be a good compromise between stability, and maintenance.

Reply:
fine by me 

Technically the exact versions would still be pinned in the package-lock.json file and so `npm ci` would ensure that the node_modules stays the same for every developer (and renovatebot is configured here to maintain the lock file weekly and so I think that the lock file would be kept updated with the last exact versions available).

Anyway I'm fine to let renovatebot to manage pinning them into the package.json file as well.

@wagnerand Feel free to merge this on master (I already changed the PR title to include the "conventional changelog" prefix in the commit message, [the convetion we are also mentioning in the contrib md file here](https://github.com/mozilla/extension-activity-monitor/blob/master/CONTRIBUTING.md#writing-commit-message), would you mind to make sure that we preserve that? it will make our life easier once we will start to put together actual changelogs for the releases, I just noticed that we missed the prefix in the last commit currently on master).

Reply:
Will do, sorry about that.

Reply:
no worries at all, we are not even enforcing it as we do in the web-ext repo and so it is definitely easy to forget (and one commit without the prefix convention at this stage is not going to be a big deal)

Reply:
Well, maybe we should start enforcing it. Ideally it would be a pre-commit hook because I find travis telling you a bit late.